src/measurements/measurements.ts

```diff
@@ -39,7 +37,10 @@ export function distance(from: Point, to: Point): number {
  * @returns distance in meters
  */
 export function geodesicDistance(from: Point, to: Point): number | undefined {
```

Comment:
Isn't `geodeticDistance` the correct name?

Reply:
it refers to the shortest path between two points which is `geodesic`

Reply:
Ok, TY.